### PR TITLE
Feedback fixes...

### PR DESCRIPTION
**Pull request by tfranjeet**
Feedback fixes....

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function login(Request $request)
                 return response()->json(['status' => 0,'message' => "Your account is in-active"]);
             }
             
-            if($user->hasRole(MEMBER_ROLE_ID) || $user->hasRole(COMPANY_ADMIN_ROLE_ID) )
+            if(!$user->hasRole(SUPER_ADMIN_ROLE_ID) )
```
Comment (gagandeepgambhir2) [inline app/Http/Controllers/UsersController.php]: Both conditions should be merged.

File: app/Http/Controllers/UsersController.php [hunk 2]
```diff
@@ -209,7 +202,9 @@ public function update(Request $request, $id)
 		
         $edit_rules = [
```
Comment (gagandeepgambhir2) [inline app/Http/Controllers/UsersController.php]: Rules should be at the top, and extend where it needs.

Comment (tfranjeet) [reply app/Http/Controllers/UsersController.php]: Fixed..

File: app/Http/Controllers/UsersController.php (hunk 1 above)
Comment (tfranjeet) [reply app/Http/Controllers/UsersController.php]: Fixed...

**Pull request merged**